analyzer/src/main/kotlin/managers/Maven.kt

```diff
@@ -176,7 +176,14 @@ class Maven(analyzerConfig: AnalyzerConfiguration, repoConfig: RepositoryConfigu
                 maven.parsePackage(node.artifact, node.repositories, localProjects, sbtMode)
             }
 
-            val dependencies = node.children.map { parseDependency(it, packages) }.toSortedSet()
+            val dependencies = node.children.mapNotNull { child ->
+                if (child.artifact.identifier().startsWith("jdk.tools:jdk.tools:")) {
+                    log.info { "Omitting the Java < 1.9 system dependency on 'tools.jar'." }
```

Comment:
Could you please explain in the commit message and code comment what's the story behind this? Why does it exist, and why only below 1.9?

Reply:
Done.